docs/alerts/index.md

```diff
@@ -228,3 +285,107 @@ An email notification hook is created automatically during signup.  Additional n
         </a>
     </div>
 </div>
+
+An email notification hook is created automatically during signup. Additional notification hooks can be created on the Notification Hooks view. A convenient way to navigate here is using the shortcut on top of the Alert Rules view.
+
+![image alt text](../images/alerts/image_10.png)
+
+It’s a shortcut for Integrations > Notification Hooks.
+
+![image alt text](../images/alerts/image_11.png)
+
+### Account-default hooks
```

Comment:
This section has a dedicated file called `account-default-hooks.md` in the same directory as this `index.md`. This section should be moved there.